specification/gedcom-3-structures-1-organization.md

```diff
@@ -396,6 +396,90 @@ However, as of 2021 relatively few applications have a user interface that prese
 A `SHARED_NOTE_RECORD` may contain a pointer to a `SOURCE_RECORD` and vice versa. Applications must not create datasets where these mutual pointers form a cycle. Applications should also ensure they can handle invalid files with such cycles in a safe manner.
 
 
+#### `SHARED_PLACE_RECORD` :=
+
+```gedstruct
+n @XREF:SPLAC@ SPLAC <Text>                {1:1}  g71:record-SPLAC
+  +1 TYPE <Text>                           {0:1}  g71:SPLAC-TYPE
+  +1 LANG <Language>                       {0:1}  g7:LANG
+  +1 TRAN <Text>                           {0:M}  g71:TRAN
+     +2 LANG <Language>                    {0:1}  g7:LANG
+  <<PLACE_DETAILS>>                        {1:1}
+  <<SHARED_PLACE_STRUCTURE>>               {0:M}  g71:SPLAC
+```
+
+A descriptor of a single place, location, or jurisdiction.
+
+The `<<SHARED_PLACE_STRUCTURE>>` inside a `SHARED_PLACE_RECORD` points to larger jurisdictions that this place is a part of.
+If a city is part of a county which is part of a state, the city's place record should point to the county's place record, not the states.
+Multiple `<<SHARED_PLACE_STRUCTURE>>`s are permitted to support places within multiple hierarchies (for example, a church that's both within an ecclesiastical region and a political region).
```

Comment:
The `HIERARCHICAL_RELATIONSHIP` is needed (per existing `_LOC` usage) inside a "shared place structure" within a `SHARED_PLACE_RECORD`, but not from within a `PLACE_REFERENCE` since the latter is not a containment relationship.  This suggests that using `<<SHARED_PLACE_STRUCTURE>>` in both places may not be appropriate.